charts/qdrant/templates/pdb.yaml

```
@@ -0,0 +1,21 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: qdrant
```

Sorry for missing this earlier. This should be:

```suggestion
  name: {{ include "qdrant.fullname" . }}
```